lectures/python_by_example.md

```diff
@@ -550,10 +551,11 @@ plt.legend()
 plt.show()
 ```
 
-Note: `f'$\\alpha = {α}$'` in the solution is an application of [f-String](https://docs.python.org/3/tutorial/inputoutput.html#tut-f-strings), which allows you to use `{}` to contain an expression. 
+```{note}
```

Comment:
@HumphreyYang I can't see this note anywhere in the output?

Reply:
@HumphreyYang so it turns out you can't seach for text in the dropdown items ... good to know.

<img width="815" alt="Screen Shot 2022-09-21 at 7 50 19 pm" src="https://user-images.githubusercontent.com/8263752/191473748-0fbd9249-a1ce-4dc1-8a36-c249699be957.png">

Reply:
> @HumphreyYang I can't see this note anywhere in the output?

@mmcky I move it down to the solution part.

Reply:
> @HumphreyYang so it turns out you can't seach for text in the dropdown items ... good to know.
> 
> <img alt="Screen Shot 2022-09-21 at 7 50 19 pm" width="815" src="https://user-images.githubusercontent.com/8263752/191473748-0fbd9249-a1ce-4dc1-8a36-c249699be957.png">

I moved it down here because it is a bit too long and it does not fit into the structure of the hint if we add another note admonition. Would you like me to move it back to the hint?